_episodes_rmd/02-func-R.Rmd

```diff
@@ -285,11 +285,11 @@ We have one more task first, though: we should write some [documentation]({{ pag
 A common way to put documentation in software is to add [comments]({{ page.root }}/reference.html#comment) like this:
 
 ```{r}
-center <- function(data, desired) {
+center <- function(data, around) {
   # return a new vector containing the original data centered around the
-  # desired value.
+  # around value.
```

Comment:
I agree that this change makes sense and I like this PR a lot. The only point of confusion is that this now reads "original data centered around the around..." which might take a moment.

Perhaps: original data centered around the value of "around" ? A few more words.

Reply:
Yeah 😅 Or `mid_point` as the parameter name?

Reply:
I like that a lot!

Reply:
Done :-) Thanks for confirming!